### DISCUSSION TITLE
Dashboard cards on Index and Show views

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#766] [Feature requests]
This is something we'd like to support ASAP!

**Comment by adrianthedev** [comment]
Working on this in https://github.com/avo-hq/avo/discussions/766

**Reply by Paul-Bob** [reply]
☝🏼 inception

**Comment by Paul-Bob** [comment]
https://docs.avohq.io/3.0/resources.html#cards
